Sources/Adapters/TFOptimizer.swift

```diff
@@ -20,7 +20,7 @@ public extension TFOptimizer {
     /// This information can later be used by the 'activationNeuron' function
     func addNeuronIfThere(node: TFNode) {
         let outgoing = node.outgoingNodes()
-        if outgoing.count == 1, let next = (outgoing.first as? TFNode),
+        if let next = (outgoing.first as? TFNode),
```

Comment:
I think we don't want to remove this check. There could be more than one outgoingNodes